articles/industry-4.0-and-cybersecurity/index.md

```diff
@@ -0,0 +1,208 @@
+
+The digital revolution, also known as the information age is dated between 1950 and 1970 and it has to be appreciated that it was a result of strides in computers and communication technology and has arguably advanced to include automation and communications since the 1970s. However, there have been no disruptive leaps forward in an industrial level since then. Rather, progress had been at the consumer level, with the ubiquity of mobile devices and emergence of e-commerce.
+
+With the above introduction to the third industrial revolution which attained automation using computer systems, comparing industry 4.0 to the third industrial revolution it can be distinguished that in the fourth and current industrial revolution, the focus of design principles is on connectivity and interconnection, information transparency, decentralization among others. 
+
+Therefore, Industry 4.0 as an initiative can be described as a new paradigm that uses various technologies to create production systems of the future. This system is meant to be automated, integrated, flexible and highly customizable. We refer to these technologies as enabling technologies. They include; Internet of Things (IoT), Cyber-Physical Systems, Smart Factory and the Cloud. We can also argue that interoperability is a key enabler of Industry 4.0 since these technologies collaborate and are integrated. There are more enabling technologies but we shall only delve into the four.
+
+
+### Industry 4.0 Enabling Technologies
+
+#### Internet of Things (IoT)
+
+The Internet of Things (IoT) refers to a network of interconnected objects. These objects are physical devices and are billions in number globally. From the term IoT, there are two parts; the internet and things. The connection is through the internet. Things are the billions of physical devices. This interconnection via the internet allows for cooperation and interaction in collecting, sharing and exchange of data. Due to cheap processors and wireless networks, it is easy to convert a physical object into part of the Internet of Things. Devices may be cars, sensors, buildings, machines and so forth. 
+
+The Internet of Things can be divided into Consumer IoT and Industrial IoT(IIoT). Consumer IoT defines smart connected product systems targeting an individual consumer. IIoT is centered around machine connectivity with an aim to not only to improve asset performance and product quality but also step up accountability and accountability throughout the systems.
```

Comment:
"but also step up accountability and accountability throughout the systems." 

just wanted to clear up what the second accountability  - was meant to say?? @collins-a

Reply:
Good eye @hectorkambow error of repetition on my part. Should be _step up accountability through the systems_